cirq/ops/common_gates_test.py

```diff
@@ -294,15 +295,17 @@ def test_text_diagrams():
         cirq.IdentityGate(2)(a,b))
 
     cirq.testing.assert_has_diagram(circuit, """
-a: ───×───X───Y───Z───Z^x───@───@───X───H───iSwap───iSwap──────I───I───
-      │                     │   │   │       │       │              │
-b: ───×─────────────────────@───X───@───────iSwap───iSwap^-1───────I───
+a: ───×───X───Y───Z───Z^Symbol("1.0*x")───@───@───X───H───iSwap───iSwap──────I───I───
```

Comment:
We shouldn't have a `1.0*` in the exponent here. This may be way the other test cases were swiched to errors? We likely need to add a case to various gate pow methods to avoid multiplying symbols by 1.